src/App.vue

```diff
@@ -246,6 +241,7 @@ export default {
 
 	created() {
 		window.addEventListener('beforeunload', this.preventUnload)
+		useHotKey('f', this.handleAppSearch, { ctrl: true, stop: true, prevent: true })
```

Comment:
Doesnt' work because event is captured by UnifiedSearch (same on master, so not related to this PR):

https://github.com/nextcloud/server/blob/b36ced878ff493cc709b0d4e8867ae9fa09c8480/core/src/views/UnifiedSearch.vue#L130-L138

Reply:
Works in the desktop client